hypha/apply/projects/migrations/0069_contractpacketfile_contractdocumentcategory.py

```diff
@@ -72,7 +72,7 @@ class Migration(migrations.Migration):
                     "document",
                     models.FileField(
                         storage=django.core.files.storage.FileSystemStorage(),
-                        upload_to=hypha.apply.projects.models.project.contract_document_path,
+                        upload_to=hypha.apply.projects.models.projects.contract_document_path,
```

Comment:
Will these create any issues with existing documents?
I checked for a few documents, seems to be working fine.